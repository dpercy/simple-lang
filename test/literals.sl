#lang s-exp "../bootstrap/sl.rkt"
;; TODO get rid of this #lang thing

1
2 3
9999999
"hello"
#true #false
