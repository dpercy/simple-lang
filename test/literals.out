1
2
3
9999999
"hello"
#true
#false
